go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -308,6 +308,8 @@ func setupOVNNode(node *kapi.Node) error {
 		fmt.Sprintf("external_ids:ovn-monitor-all=%t", config.Default.MonitorAll),
 		fmt.Sprintf("external_ids:ovn-ofctrl-wait-before-clear=%d", config.Default.OfctrlWaitBeforeClear),
 		fmt.Sprintf("external_ids:ovn-enable-lflow-cache=%t", config.Default.LFlowCacheEnable),
+		// when creating tunnel ports set local_ip, helps ensures multiple interfaces and ipv6 will work
+		"external_ids:ovn-set-local-ip=\"true\"",
```

Comment:
@JacobTanenbaum I see:
```
external_ids:ovn-set-local-ip
                     The boolean flag indicates if ovn-controller when  create
                     tunnel  ports should set local_ip parameter. Can be hepl‐
                     ful to pin source outer IP for the tunnel  when  multiple
                     interfaces are used on the host for overlay traffic. This
                     is  also  useful when running multiple ovn-controller in‐
                     stances on the same chassis, in which case  this  setting
                     will  guarantee  that their tunnel ports have unique con‐
                     figuration and can exist in parallel.
```

1. when  multiple interfaces are used on the host for overlay traffic. => we don't do this..
2. This is  also  useful when running multiple ovn-controller instances on the same chassis => we don't do this too
3. finally the stupid question ; is this inferring a pinning to primary ip somehow? we are just passing a boolean here right? how does it pick which IP to pin to?

Reply:
got questions answered in slack all good.